integration_tests/run_pyspark_from_build.sh

```diff
@@ -42,8 +42,16 @@ else
         CUDF_JARS=$(echo "$LOCAL_JAR_PATH"/cudf-*.jar)
         AVRO_JARS=$(echo "$LOCAL_JAR_PATH"/spark-avro*.jar)
         PLUGIN_JARS=$(echo "$LOCAL_JAR_PATH"/rapids-4-spark_*.jar)
+
+        INTEGRATION_TEST_VERSION=$SPARK_SHIM_VER
+
+        if [[ ! -z $INTEGRATION_TEST_VERSION_OVERRIDE ]]; then
```

Comment:
please move this up to above line 41, it can be outside the if [ -d $local_jar_path ] check and be able to apply to both this section and else section.

Reply:
OK

Reply:
refined